### PR TITLE
Add mount command

### PR DESCRIPTION
**Pull request by alaypatel07**
This enables us to use CEO image for kube client agent for CEO managed etcd pods.

Depends on #46

### CODE REVIEW

**Comment by alaypatel07** [comment]
@hexfusion I will rebase this once we merge #46

**Comment by hexfusion** [comment]
I would like to merge this while we talk about #46 can you drop it from this PR, please.

File: go.mod [hunk 1]
```diff
@@ -35,6 +35,7 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-openapi/spec v0.19.2 // indirect
 	github.com/gogo/protobuf v1.2.1 // indirect
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
 	github.com/golang/groupcache v0.0.0-20190129154638-5b532d6fd5ef // indirect
```
Comment (hexfusion) [inline go.mod]: we have klog right do we need this.

Comment (alaypatel07) [reply go.mod]: we already have klog, dropping this change. good catch, +1

File: pkg/cmd/mount/mount.go [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+		},
+	}
+
+	mountSecretOpts.AddFlags(cmd.Flags())
+	return cmd
+}
+
+func (m *mountSecretOpts) AddFlags(fs *pflag.FlagSet) {
+	fs.StringVar(&m.commonName, "commonname", "", "Common name for the certificate being requested")
+	fs.StringVar(&m.assetsDir, "assetsdir", "", "Directory location for the agent where it stores signed certs")
```
Comment (hexfusion) [inline pkg/cmd/mount/mount.go]: nit: `Directory location to store signed certs` ?

**Pull request merged**